docs/content/en/docs/workshops/provision/local_cluster.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Local/Dev Cluster Setup"
```

Comment:
We try to stay consistent with calling the docker provisioner a "local" cluster because dev could mean a lot of things and could be hosted on vSphere etc
```suggestion
title: "Local cluster setup"
```